.circleci/config.yml

```diff
@@ -433,6 +480,23 @@ workflows:
             tags:
               only: /.*/
 
+      # Internationalization jobs
+      #
+      # Extract strings and upload them to our translation management platform
+      - build-i18n:
+          requires:
+            - build-back
+          filters:
+            tags:
+              only: /.*/
+      - upload-i18n-strings:
+          requires:
+            - build-back
```

Comment:
this is unecessary since build-i18n depends from build-back.

Reply:
Indeed !